packages/app/comments/service.ts

```diff
@@ -21,18 +21,17 @@ export async function createCommentAsUser(
     try {
         const commentsDb = await getCommentsDb()
 
-        if (!user?.uid) {
-            throw new HttpException(ErrorCode.Unauthorized, 'Unauthorized')
-        }
+        assert(user?.uid, new createError.Unauthorized())
```

Comment:
Again, I won't comment every assert, but introducing "assert" to clean up some of these conditions